docs/style-guide.adoc

```diff
@@ -1,6 +1,14 @@
+:numbered:
```

Comment:
It does look a bit awkward, though...and they're both very long sections as well. Maybe they warrant being "pulled out", just like "Graph modelling" is for the Recommendations section?